src/renderer/components/+extensions/extensions.tsx

```diff
@@ -187,15 +187,17 @@ export class Extensions extends React.Component {
     await Promise.all(
       requests
         .filter(req => !req.data && req.filePath)
-        .map(request => {
-          return fse.readFile(request.filePath).then(data => {
+        .map(async request => {
+          try {
+            const data = fse.readFileSync(request.filePath);
```

Comment:
I don't know why, but using `readFileSync` is notably faster on Windows

Reply:
I don't think sync methods should be used in the UI, since they lock the UI. (No idea why it would be faster).

Reply:
I, personally, take that a step further, I don't think file IO at all should be done in the renderer.

Reply:
Agree, I think we should start to move "non-UI" code to "services".

Reply:
Got it working with `readFile`.